androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/playerOverlay/BottomControls.kt

```diff
@@ -48,17 +48,17 @@ fun BottomControls(
             .padding(Dimensions.small)
     ) {
         Slider(
-            value = currentTime.toFloat(),
+            value = currentTime().toFloat(),
```

Comment:
You can make the slider skips recomposition entirely by wrapping the `currentTime().toFloat()` by using:

```kotlin
    val currentTimeInFloat by rememberSaveable { mutableStateOf(currentTime().toFloat()) }
```

And

```suggestion
            value = currentTimeInFloat,
```

Reply:
I think it doesn't need to be a `MutableState`, since it won't be changed in this composable scope. And don't forget to add a key on `rememberSaveable` to update its value whenever the parameter changes.